### ISSUE TITLE
Can't Import Tensor Flow in Annaconda Python 3.6

### ISSUE THREAD

**Issue by cpodczerwinski** [tensorflow/tensorflow#20584]
I just installed CUDA 9.2 and CUDANN and Tensor Flow on Windows 10.  I'm using Anaconda 3.6 Python.

Here is a trace that I get when I try to import tensor flow.  It's telling me it can't load a dll but its is not telling me which one.  Can you help.  

Here is a directory listing of the CUDA dlls in my path and the trace.  

********************  Directory Listing ************************

PS C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v9.2\bin> dir

    Directory: C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v9.2\bin

Mode                LastWriteTime         Length Name
----                -------------         ------ ----
d-----         7/5/2018   8:32 PM                crt
-a----        4/12/2018   1:20 AM         202752 bin2c.exe
-a----        4/12/2018   1:21 AM       54600704 cublas64_92.dll
-a----        4/12/2018   1:20 AM         370176 cuda-memcheck.exe
-a----        4/12/2018   1:20 AM        4092416 cudafe++.exe
-a----        4/12/2018   1:21 AM         299008 cudart32_92.dll
-a----        4/12/2018   1:21 AM         368128 cudart64_90.dll
-a----        4/12/2018   1:21 AM         368128 cudart64_92.dll
-a----         7/5/2018  10:17 PM      336443392 cudnn64_7.dll
-a----        4/12/2018   1:21 AM       87017472 cufft64_92.dll
-a----        4/12/2018   1:21 AM         197632 cufftw64_92.dll
-a----        4/12/2018   1:21 AM        3692032 cuinj32_92.dll
-a----        4/12/2018   1:21 AM        4625408 cuinj64_92.dll
-a----        4/12/2018   1:20 AM        1695744 cuobjdump.exe
-a----        4/12/2018   1:21 AM       47990784 curand64_92.dll
-a----        4/12/2018   1:21 AM      114172416 cusolver64_92.dll
-a----        4/12/2018   1:21 AM       65776128 cusparse64_92.dll
-a----        4/12/2018   1:20 AM         284672 fatbinary.exe
-a----        4/12/2018   1:20 AM        1306112 gpu-library-advisor.exe
-a----        4/12/2018   1:21 AM         203264 nppc64_92.dll
-a----        4/12/2018   1:21 AM       10174464 nppial64_92.dll
-a----        4/12/2018   1:21 AM        3985920 nppicc64_92.dll
-a----        4/12/2018   1:21 AM        1010688 nppicom64_92.dll
-a----        4/12/2018   1:21 AM        6928896 nppidei64_92.dll
-a----        4/12/2018   1:21 AM       51902976 nppif64_92.dll
-a----        4/12/2018   1:21 AM       25260032 nppig64_92.dll
-a----        4/12/2018   1:21 AM        6574592 nppim64_92.dll
-a----        4/12/2018   1:21 AM       15030272 nppist64_92.dll
-a----        4/12/2018   1:21 AM         177152 nppisu64_92.dll
-a----        4/12/2018   1:21 AM        2621440 nppitc64_92.dll
-a----        4/12/2018   1:21 AM        8543232 npps64_92.dll
-a----        4/12/2018   1:20 AM         241152 nvblas64_92.dll
-a----        4/12/2018   1:20 AM         379904 nvcc.exe
-a----        4/12/2018   1:20 AM            310 nvcc.profile
-a----        4/12/2018   1:20 AM       18156032 nvdisasm.exe
-a----        4/12/2018   1:21 AM       66096128 nvgraph64_92.dll
-a----        4/12/2018   1:20 AM        7804928 nvlink.exe
-a----        4/12/2018   1:20 AM        4115968 nvprof.exe
-a----        4/12/2018   1:20 AM         220160 nvprune.exe
-a----        4/12/2018   1:20 AM        3213312 nvrtc-builtins64_92.dll
-a----        4/12/2018   1:20 AM       15532544 nvrtc64_92.dll
-a----        4/12/2018   1:20 AM             53 nvvp.bat
-a----        4/12/2018   1:20 AM        7686144 ptxas.exe

********************  Python Output  ************************

PS C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v9.2\bin> python
Python 3.6.0 |Anaconda 4.3.0 (64-bit)| (default, Dec 23 2016, 11:57:41) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import tensorflow as tf
Traceback (most recent call last):
  File "C:\Program Files\Anaconda3\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 58, in <module>
    from tensorflow.python.pywrap_tensorflow_internal import *
  File "C:\Program Files\Anaconda3\lib\site-packages\tensorflow\python\pywrap_tensorflow_internal.py", line 18, in <module>
    _pywrap_tensorflow_internal = swig_import_helper()
  File "C:\Program Files\Anaconda3\lib\site-packages\tensorflow\python\pywrap_tensorflow_internal.py", line 17, in swig_import_helper
    return importlib.import_module(mname)
  File "C:\Program Files\Anaconda3\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ImportError: DLL load failed: The specified module could not be found.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Program Files\Anaconda3\lib\site-packages\tensorflow\__init__.py", line 24, in <module>
    from tensorflow.python import pywrap_tensorflow  # pylint: disable=unused-import
  File "C:\Program Files\Anaconda3\lib\site-packages\tensorflow\python\__init__.py", line 49, in <module>
    from tensorflow.python import pywrap_tensorflow
  File "C:\Program Files\Anaconda3\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 74, in <module>
    raise ImportError(msg)
ImportError: Traceback (most recent call last):
  File "C:\Program Files\Anaconda3\lib\site-packages\tensorflow\python\pywrap_tensorflow.py", line 58, in <module>
    from tensorflow.python.pywrap_tensorflow_internal import *
  File "C:\Program Files\Anaconda3\lib\site-packages\tensorflow\python\pywrap_tensorflow_internal.py", line 18, in <module>
    _pywrap_tensorflow_internal = swig_import_helper()
  File "C:\Program Files\Anaconda3\lib\site-packages\tensorflow\python\pywrap_tensorflow_internal.py", line 17, in swig_import_helper
    return importlib.import_module(mname)
  File "C:\Program Files\Anaconda3\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ImportError: DLL load failed: The specified module could not be found.

Failed to load the native TensorFlow runtime.

See https://www.tensorflow.org/install/install_sources#common_installation_problems

for some common reasons and solutions.  Include the entire stack trace
above this error message when asking for help.
>>>

**Comment by ireqhawk** [comment]
> File "C:\Program Files\Anaconda3\lib\importlib_init.py", line 126, in import_module
return _bootstrap._gcd_import(name[level:], package, level)
ImportError: DLL load failed: The specified module could not be found
maybe you can just add print(name[leval:]) in importlib_init.py to locate which dll load failed.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @shivaniag: It has been 17 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by cpodczerwinski** [comment]
Thank you for getting back to me. I found that the problem was caused by the fact that I installed the latest version of the CUDA library. I downgraded to one version earlier and everything worked. 
RegardsCraig


Sent from Yahoo Mail for iPhone


On Sunday, July 29, 2018, 5:25 PM, Gunhan Gulsoy <notifications@github.com> wrote:


What is your CPU model? I suspect this may be a duplicate of #19584
